arc.arc

```diff
@@ -648,6 +648,10 @@ More convenient form of [[coerce]] with arguments reversed; doesn't need
 "Converts 'x' into a symbol."
   (coerce x 'sym))
 
+(def str (x)
```

Comment:
Do we need this, when we have [`string`](https://github.com/arclanguage/anarki/blob/cd574e3bd2a88d509f0d49d6eb06cf85d6101bed/arc.arc#L659)?

Reply:
Possibly. `(coerce x 'string)` behaves differently from `(+ "" x)`.

Reply:
In what situation?

Reply:
In no situations. Never mind, I was mistaken. :)

One benefit of adding a `str` function is that it can be a place to do things like printing a hash table using curly brace syntax. But that's work for some other time. I'll get rid of `str`.

Reply:
Also, feel free to overload `string` for those purposes.

(I don't particularly what the function is called. But it seems good to have one strong default.)